lib/livebook/hubs/dockerfile.ex

```diff
@@ -421,16 +396,16 @@ defmodule Livebook.Hubs.Dockerfile do
                 ~s{<a class="font-medium underline text-gray-900 hover:no-underline" href="https://livebook.dev/teams?ref=LivebookApp" target="_blank">Livebook Teams</a>}
 
               "This app has no password configuration and anyone with access to the server will be able" <>
-                " to use it. You may either configure a password or use #{teams_link} to add Zero Trust Authentication" <>
+                " to use it. You may either configure a password or use #{teams_link} to enable Livebook Teams authentication" <>
                 " to your deployed notebooks."
```

Comment:
Let's just link to the authentication page instead. Something like:

`See the documentation on <a href="...">Authentication</a> for more information.`